code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -8,7 +8,8 @@
 	var/t_is = p_are()
 
 	var/msg = "<span class='info'>*---------*\nThis is <EM>[src.name]</EM>!\n"
-
+	if(isdwarf(src))

```

Comment:
Could this not be a function of species datum in order to avoid checking every single human mob during examine()?